app/bundles/CoreBundle/Loader/ParameterLoader.php

```diff
@@ -82,29 +82,35 @@ public function loadIntoEnvironment()
         $dotenv->populate($envVariables->all());
     }
 
-    public static function getLocalConfigFile(string $root): string
+    public static function getLocalConfigFile(string $root, $updateDefaultParameters = true): string
```

Comment:
`$updateDefaultParameters` is needed because once one or more keys are set in `$defaultParameters`, the `loadDefaultParameters()` function doesn't work anymore because it thinks that all default parameters have already been set:

https://github.com/mautic/mautic/blob/c8061156356c0edd3bca09e2c54f62215d400fd0/app/bundles/CoreBundle/Loader/ParameterLoader.php#L110-L116

Since we only access `getLocalConfigFile()` in the `AppKernel`, and not any other config files or env variables, we want to prevent `$defaultParameters` from being set incompletely on application boot. Therefore we explicitly set `$updateDefaultParameters` to `false` when accessing this function from the AppKernel.